packages/declaration/src/objectData.d.ts

```diff
@@ -8,7 +8,7 @@ interface ObjectData {
   GUID?: GUID;
 
   /** The type of the object. */
-  Name: string;
+  Name: ObjectType;
```

Comment:
It should be `ObjectName` instead. `ObjectType` is the readable information that you get, when using `.type` on an actual object. The name in the object data is way more convuluted. ^^" E.g. the `Name` could be `Custom_Model_Bag` for a custom bag, but the `type` would be `Bag` just like the regular TTS bag.

Reply:
Whoops. Good catch. I have no idea how I managed that. I had it fixed the correct way when I changed it in node modules for my other project. I had some struggles with learning how to update the PR so I hope the git history is fine.